lib/browser.ts

```diff
@@ -188,7 +195,21 @@ export class ProtractorBrowser extends Webdriver {
    *
    * @type {boolean}
    */
-  ignoreSynchronization: boolean;
+  set ignoreSynchronization(value) {
```

Comment:
Good point. I'll add the new API, but I'd like to leave `ignoreSynchronization` in place and announce that it's deprecated.